schema_based_on_diagram

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE patients(
+    id INTEGER NOT NULL,
+    name VARCHAR(150) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    PRIMARY KEY(id)
+);
+
```

Comment:
- [ ] In your project folder, I cannot find the correct diagram. Please kindly update with the diagram in the screenshot below.
![image](https://user-images.githubusercontent.com/62849872/211765268-64b7db9c-a2f7-41dd-9526-04317b1ec994.png)
- Below is what I can see in your project folder
![image](https://user-images.githubusercontent.com/62849872/211765499-63369a8a-16a2-4960-9edf-b91cfc01a34b.png)